### ISSUE TITLE
Implement new top nav

### ISSUE THREAD

**Issue by arielkennan** [codeforamerica/bizfriendly-web#63]
**Main Nav Bar**
On hover: text turns blue and icon (20px x 20 px) appears above
On click: white bar below section, icon remains

**Sub Nav**
On hover: light orange
On click: orange bar below

![bizfriendly_newnav_090313](https://f.cloud.github.com/assets/3219885/1076569/a6ee5bfe-14e4-11e3-83c7-9ecdd1eb655d.gif)
![connect_icon_gray20px](https://f.cloud.github.com/assets/3219885/1076572/b0ef3736-14e4-11e3-96b5-a887049e18fe.gif)
![learnicon_gray20px](https://f.cloud.github.com/assets/3219885/1076587/11536de0-14e5-11e3-9660-c1e3c97a3b55.gif)
![teachicon_gray20px](https://f.cloud.github.com/assets/3219885/1076588/146d6ee0-14e5-11e3-943e-1aefa3fec8ed.gif)

**Comment by nickjellings** [comment]
See #182 for implementation of navigation active & hover states.

**Comment by ondrae** [comment]
Check it out @arielkennan. Another great addition by @nickjellings 
Any tweaks you want to see?

**Comment by arielkennan** [comment]
Nice work @nickjellings! 

This was a fairly old issue and probably should have been updated to no longer include the icons. We had used those for 90% of the way through the design, and then updated them to be more of what's on the homepage now. I'm not sure we need them anymore in the nav. Sorry I hadn't updated this issue as we had iterated on the design. 

The underlines look great, however!!! 

One bug, Teach seems to be staying highlighted no matter what? Checked both in Chrome and Safari.
